backend/lib/endpoints/schema/geo.js

```diff
@@ -21,8 +27,22 @@ const getLocationReverseGeocodeSchema = {
     .prop("lng", S.number().minimum(-180).maximum(180).required()),
 };
 
+const getHealthFacilityPlacesSchema = {
+  querystring: strictSchema()
+    .prop("lat", S.number().minimum(-90).maximum(90).required())
+    .prop("lng", S.number().minimum(-180).maximum(180).required())
+    .prop(
+      "type",
+      S.string()
+        .enum(Object.values(HEALTH_FACILITY_TYPES))
+        .default(HEALTH_FACILITY_TYPES.HOSPITAL),
+    ),
+};
```

Comment:
if the response format is finalized, we may add the response schema inside, e.g. 
```
  response: {
    200: S.object()
      .prop(...
  },
```
more examples in this [link](https://github.com/FightPandemics/FightPandemics/pull/2049/files)

Reply:
@joshmorel may have thoughts on this, since he's already code-reviewed this.  Thanks.